vsts/windowsLinuxAndAndroidBuildMatrixConfig.yaml

```diff
@@ -173,7 +173,7 @@ jobs:
       targetPath: 'iot-e2e-tests/android/app/build/outputs/apk'
     
 - job: AndroidTest
-  timeoutInMinutes: 180
+  timeoutInMinutes: 80
```

Comment:
is this timeout waiting for agent?

Reply:
This is a timeout for waiting for the agent + running the tests. So if it takes more than 80 minutes to get an Android agent, the task will fail

Reply:
This will also help to cut down on cases where the java builds hog azure devops agents when no app center agents are available